interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>These two paragraphs are at the right level of detail (don't go into the physical motivations of flare shapes). I think some external links would be good here (e.g. to an explanation of gradient descent)</p><p> </p><p>Describing CNNs here would be the right move</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>